week-1/Homework/mandatory/1-syntax-errors.js

```diff
@@ -1,18 +1,22 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+function addNumbers(a, b, c) {
     return a + b + c;
 }
 
-function introduceMe(name, age)
-return "Hello, my name is " + name "and I am " age + "years old";
+function introduceMe(name, age) {
+return "Hello, my name is " + name + "and I am" + age + "years old";
+}
 
 function getRemainder(a, b) {
-    remainder = a %% b;
+    remainder = a % b;
+}
+
+
 
     // Use string interpolation here
-    return "The remainder is %{remainder}"
-}
+    return `The remainder is ${getRemainder}`
```

Comment:
Replacing the quotes with ticks and % with $ is right, but the return should be inside the function, and the variable name is remainder.